Release.toml

```diff
@@ -357,3 +357,6 @@ version = "1.24.1"
     "migrate_v1.24.1_aws-control-container-v0-7-16.lz4",
     "migrate_v1.24.1_public-control-container-v0-7-16.lz4",
 ]
+"(1.24.1, 1.25.0)" = [
```

Comment:
You shouldn't refer to a version that is higher than the on at the top of this file, you should bump that to `1.25.0` as well in this change.